.circleci/config.yml

```diff
@@ -39,6 +39,8 @@ jobs:
     docker:
       - image: cimg/node:14.20
     steps:
+      - run: 
```

Comment:
npm workspaces don't work on the version of npm that is installed on the build image. We need to upgrade it. Thankfully this only adds ~10s to the build.